README.rst

```diff
@@ -7,7 +7,12 @@ An `open-source <https://github.com/aiven/karapace/blob/master/LICENSE>`_ implem
 of `Kafka REST <https://docs.confluent.io/platform/current/kafka-rest/index.html#features>`_ and
 `Schema Registry <https://docs.confluent.io/platform/current/schema-registry/index.html>`_.
 
-.. image:: https://github.com/aiven/karapace/actions/workflows/tests.yml/badge.svg
+|Tests| |Contributor Covenant|
+
+.. |Tests| image:: https://github.com/aiven/karapace/actions/workflows/tests.yml/badge.svg
+
+.. |Contributor Covenant| image:: https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg
```

Comment:
nit pick: Name this "Contributor Covenant Code of Conduct" or "Contributor Covenant CoC"?